library/rtsp/src/main/java/com/google/android/exoplayer2/source/rtsp/RtspMediaSource.java

```diff
@@ -117,6 +119,19 @@ public Factory setDebugLoggingEnabled(boolean debugLoggingEnabled) {
       return this;
     }
 
+    /**
+     * Configures a socket factory to be used for client connections.
```

Comment:
Nit: maybe 

Sets a socket factory for {@link RtspClient}'s connection, the default value is {@link SocketFactory#getDefault()}.

And remove the following line